src/apprt/gtk/notebook.zig

```diff
@@ -1,169 +1,193 @@
+/// An abstraction over the GTK notebook and Adwaita tab view to manage
+/// all the terminal tabs in a window.
+const Notebook = @This();
```

Comment:
Instead of notebook which is GTK parlance. Let's just use TabView. Maybe do this in a separate commit to help with diff size.

Reply:
+1 on keeping diff size down.

Reply:
Even better, let's make it a separate PR. Maybe at the same time switch to zig-gobject.